2019/python/lar_constraints.py

```diff
@@ -485,11 +485,11 @@ def v648_const(self, row):
 		"""1) If Sex of Co-Applicant or Co-Borrower Collected on the Basis of Visual Observation or Surname equals 2,
 			then Sex of Co-Applicant or Co-Borrower must equal 1, 2, 3 or 6.
 		2) If Sex of Co-Applicant or Co-Borrower equals 6, then Sex of Co-Applicant or Co-Borrower Collected on the
-			Basis of Visual Observation or Surname must equal 2."""
+			Basis of Visual Observation or Surname must equal 2 or 3."""
 		if row["co_app_sex_basis"] == "2" and row["co_app_sex"] not in ("1", "2", "3", "6"):
 			row["co_app_sex"] = random.choice(("1", "2", "3", "6"))
 		if row["co_app_sex"] == "6":
```

Comment:
Please make a comment that code 3 is drawn from platform changes.

https://github.com/cfpb/hmda-platform/issues/2774

Reply:
Change has been implemented in recent commit.